src/main/java/helpers/Utilities.java

```diff
@@ -14,6 +14,7 @@
  */
 public class Utilities {
   public static final String APP_FILE = "./app.properties";
+  public static final String REG_FILE = "./register.properties";
```

Comment:
This file is being used only in the RegisterTest. You can move it (along with its getPropertyFromRegProp()) in the test class. Here, in the Utilities class (which is a framework-wade class) we implement only what is used in multiple places.
So, since the REG_FILE is used/will be used only by the RegisterTest you can move it there.

Reply:
I moved the getPropertyFromRegProp() method  and the REG_FILE variable from Utilities class to the RegisterTest class as requested.